crates/rune-modules/src/http.rs

```diff
@@ -177,6 +1197,37 @@ impl Response {
         let inner = self.response.status();
         StatusCode { inner }
     }
+
+    /// Get the version of the response.
+    #[rune::function]
+    fn version(&self) -> Version {
+        let inner = self.response.version();
+        Version { inner }
+    }
+
+    // /// Get the headers of the response.
+    // #[inline]
+    // fn headers(&self) ->  {
+    //     self.response.headers()
+    // }
+
+    /// Get the content-length of this response, if known.
+    ///
+    /// Reasons it may not be known:
+    ///
+    /// - The server didn't send a `content-length` header.
+    /// - The response is compressed and automatically decoded (thus changing
+    ///   the actual decoded length).
+    #[rune::function]
+    fn content_length(&self) -> Option<u64> {
+        self.response.content_length()
+    }
+
+    #[rune::function]
+    // TODO: create socket addr/ip addr into rune core
```

Comment:
What if we had `socket/ip addr` in the core? Or perhaps a new net module?

Reply:
Yeah. These should probably go into `std` which are the modules defined in the `rune` crate.

Reply:
I'll put this on hold for now and focus on a new crate...  I'm considering `rune/modules/net`. After that, I'll implement `remote_addr`.

We could add support for both, `std::net` and `tokio::net`. What do you think?

Reply:
I am implementing the net module in https://github.com/rune-rs/rune/pull/820, initially with IpAddr and SocketAddr.